interfaces/builtin/home.go

```diff
@@ -74,6 +74,11 @@ owner @{HOME}/snap/ r,
 # files; only allow writes to files, not the mount point).
 owner /run/user/[0-9]*/gvfs/{,**} r,
 owner /run/user/[0-9]*/gvfs/*/**  w,
+
+# On some systems HOME/bin is in the default path in ahead of system
+# directories. To prevent snaps from writing arbitrary executables waiting to
+# be launched by the user, deny writes to the directory altogether.
```

Comment:
The fact that $HOME/bin is ahead of system paths makes it particularly bad, but isn't the only reason for the change. I suggest:
```
  # Disallow writes to the well-known directory included in
  # the user's PATH on several distributions
```